client/my-sites/checkout/checkout-thank-you/remove-after-traslated/new-congrats-translations.jsx

```diff
@@ -0,0 +1,196 @@
+import { translate } from 'i18n-calypso';
+
+const translations = [
+	// Paid theme
+	translate( 'Unveil the wow factor' ),
+	translate(
+		`All set! Activate the %(themeName)s theme and take your site's style to the next level.`,
+		{
+			args: {
+				themeName: 'Livro',
+			},
+		}
+	),
+	translate( '%(themeName)s theme', {
+		args: {
+			themeName: 'Livro',
+		},
+	} ),
+	translate( 'Available until %(date)s', {
+		args: {
+			savingsValue: 'June 30, 2022',
+		},
+	} ),
+	translate( 'Activate theme' ),
+	translate( 'Need help setting up your theme?' ),
+	translate(
+		'Check out our support documentation for step-by-step instructions and expert guidance on your theme set up.'
+	),
+	translate( '{{a}}Get set up support{{/a}}', {
+		components: {
+			a: <a href=" https://wordpress.com/support/themes/set-up-your-theme/" />,
+		},
+	} ),
+	translate( 'Your go-to theme resource' ),
+	translate(
+		'Take a look at our comprehensive support documentation and learn more about themes.'
+	),
+	translate( '{{a}}Learn more about themes{{/a}}', {
+		components: {
+			a: <a href="https://wordpress.com/support/themes/" />,
+		},
+	} ),
+
+	// Free theme
+	translate( 'Way to make an impression' ),
+	translate( 'Your site looks stunning with its new theme. Take a look or start styling it up.' ),
+	translate( 'Customize the theme' ),
+	translate( 'Visit site' ),
+	translate( 'Solve anything with yout go-to theme resource' ),
```

Comment:
I noticed this change in the final version of the Figma (VLoPHWqcewxNKZYjjkDu3O-fi-2414_3286)
```suggestion
	translate( 'Your go-to theme resource' ),
```

Reply:
Thanks for catching this!
FYI there are two versions
1. Your go-to theme resource
2. Solve anything with your go-to theme resource